common/lib/common-utils/src/base64Encoding.ts

```diff
@@ -24,3 +24,9 @@ export const toUtf8 = (input: string, encoding: string): string => {
             return IsoBuffer.from(input, encoding).toString();
     }
 };
+
+/**
+ * Convert base64 or utf8 string to array buffer
+ */
+export const base64ToArrayBuffer = (input: string, encoding: string): ArrayBufferLike =>
```

Comment:
Name does not correspond to actual logic. It can accept any formatting, not just base64.
Either make name more generic, or create two functions that do not take encoding as argument - base64ToArrayBuffer  & stringToArrayBuffer.
Also, for convenience (and depending on usage), you might want to have bufferFromIBlob() kind of API